modules/Campaigns/WizardNewsletter.php

```diff
@@ -629,7 +640,7 @@ function validate_wiz_form(step){
 
 $ss->display(file_exists('custom/modules/Campaigns/tpls/WizardNewsletter.tpl') ? 'custom/modules/Campaigns/tpls/WizardNewsletter.tpl' : 'modules/Campaigns/tpls/WizardNewsletter.tpl');
 
-if(!$focus->id) {
+if(!$focus->id && isset($campaign_id) && $campaign_id) {
```

Comment:
Could we add a comment explaining why we need this code? For example What is check looking for and why do we need to unset this value?